CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+## Changelog
```

Comment:
You don't have to change this file. It will be updated by the Alpakka team whenever they release.

Reply:
Ops! I did it in order to document changes occurred, I'll remove immediately, sorry :-)